File: Python/8py 4/to_binary.py
"""
|--------------------------------------------------------------------------
| Convert to Binary
|--------------------------------------------------------------------------
|
"""

def to_binary(n):
   return int(bin(n)[2:])


def to_binary(n):
   res, count = 0, 0
   while n:
      res += n%2 * 10**count
      count += 1
      n //= 2
   return res


def to_binary(n):
   return int(f'{n:b}')


def to_binary(n):
   return int(format(n, 'b'))


print(to_binary(1), 1)
print(to_binary(2), 10)
print(to_binary(3), 11)
print(to_binary(5), 101)

File: JavaScript/8js 2/toBinary.js
/*
|--------------------------------------------------------------------------
| Convert to Binary
|--------------------------------------------------------------------------
|
*/

function toBinary(n) {
   return Number(n.toString(2));
}


function toBinary(n) {
   let arr = [];
   for (let i = n; i >= 1; i = Math.floor(i / 2)) {
      arr.push(i % 2 === 0 ? 0 : 1);
   }
   return Number(arr.reverse().join(""));
}


function toBinary(n) {
   return +n.toString(2);
}


console.log(toBinary(1), 1);
console.log(toBinary(2), 10);
console.log(toBinary(3), 11);
console.log(toBinary(5), 101);